tests/drivers/i2c/i2c_slave_api/README.txt

```diff
@@ -0,0 +1,29 @@
+I2C Slave API test
+##################
+
+This test verifies I2C slave driver implementations using two I2C
+controllers on a common bus.  The test is supported by a test-specific
+driver that simulates an EEPROM with an I2C bus follower ("slave")
+interface.  Data is pre-loaded into the simulated devices outside the
+I2C API, and the Zephyr application issues commands to one controller
+that are responded to by the simulated EEPROM connected through the
+other controller.
+
+In slightly more detail the test has these phases:
+
+* Use API specific to the simulated EEPROM to pre-populate the simulated
+  devices with device-specific content.
+* Register each simulated EEPROM as a I2C follower device on a bus.
+* Issue commands on one bus controller (operating as the bus leader
+  (master)) and verify that the data supplied by the other controller
+  (as bus follower) match the expected values given the content known to
+  be present on the simulated device.
+
+Transfer of commands from one bus controller to the other is
+accomplished by hardware through having the SCL (and SDA) signals
+shorted to join the two buses.
+
+Presence of this required hardware configuration is identified by the
+`i2c_bus_short` fixture.  If the buses are not connected as required,
```

Comment:
> `i2c_bus_short` fixture.

Shouldn't it be filtered in test yaml file then?

Reply:
Yes; apparently I didn't save those changes file so it was only present as an emacs backup.